src/main/java/ca/carleton/models/Customer.java

```diff
@@ -1,25 +1,57 @@
 package ca.carleton.models;
 
 import javax.persistence.Entity;
-import java.sql.Date;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
 
 @Entity
 public class Customer extends User {
-    private Date accessExpiration;
+    private String accessExpiration;
```

Comment:
This should stay a date, how will you check if the trial has expired with a string?